Index length too long for MySQL database.

When using MySQL version 5.7, hibernate from `catalog` triggers the following error during startup:

```java
org.hibernate.tool.schema.spi.CommandAcceptanceException: Unable to execute command [create index IDX86enfx0lwjc4ygjei28279a8 on METADATA_KEY_VALUE (PA_VALUE)]
Caused by: java.sql.SQLSyntaxErrorException: Specified key was too long; max key length is 3072 bytes
```
From [KeyValueLabelMetadataEntity](https://github.com/ow2-proactive/catalog/blob/master/src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java), the index length is indeed too long as both `PA_KEY` and `PA_VALUE` columns are indexed and the size of `PA_VALUE` itself is fixed to 10KB (see https://github.com/ow2-proactive/catalog/issues/78 for more details).

--- a/src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java
+++ b/src/main/java/org/ow2/proactive/catalog/repository/entity/KeyValueLabelMetadataEntity.java
@@ -59,8 +59,7 @@ import lombok.NoArgsConstructor;
 @NoArgsConstructor
 @Table(name = "METADATA_KEY_VALUE", uniqueConstraints = @UniqueConstraint(columnNames = { "CATALOGOBJECTREVISION",
                                                                                           "PA_KEY",
-                                                                                          "LABEL" }), indexes = { @Index(columnList = "PA_KEY"),
-                                                                                                                  @Index(columnList = "PA_VALUE") })
+                                                                                          "LABEL" }), indexes = { @Index(columnList = "PA_KEY") })
 @Inheritance(strategy = InheritanceType.SINGLE_TABLE)
 public class KeyValueLabelMetadataEntity implements Serializable {
 

